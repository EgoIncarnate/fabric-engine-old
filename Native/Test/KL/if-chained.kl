function entry()
{
  if ( false )
    report "BAD 1";
  else if ( false )
    report "BAD 2";
  else report "GOOD";
}
