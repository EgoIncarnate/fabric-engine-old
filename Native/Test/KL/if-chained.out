GlobalList
  Function( 'entry' ('entry'), void )
    ParamList
    CompoundStatement
      StatementList
        ConditionalStatement
          ConstBoolean( false )
          Report
            ConstString( 'BAD 1' )
          ConditionalStatement
            ConstBoolean( false )
            Report
              ConstString( 'BAD 2' )
            Report
              ConstString( 'GOOD' )
GOOD
